apps/web/package.json

```diff
@@ -64,6 +64,9 @@
     "@tanstack/react-query": "^4.28.0",
     "@vanilla-extract/next-plugin": "^2.3.0",
     "@wallchain/sdk": "^0.6.8",
+    "@walletconnect/notify-client": "0.14.4",
```

Comment:
I assume this is for server? If we only use type from this package let's move to devDependencies

Reply:
no i use it for types in a few components

Reply:
if only use for types then move it into devDependencies